### PR TITLE
fix(build): Move '@types/react' to devDependencies

### LINKED ISSUE #21: Build: Move `"@types/react"` to `devDependencies`
Currently, `"@types/react"` is listed as a dependency in `package.json`. To align with good development practices, we should move `"@types/react"` to `devDependencies`.

**Steps:**
1. Open `package.json` file.
2. Locate the `"dependencies"` section.
3. Find the entry for `"@types/react"` and cut it.
4. Scroll down to the `"devDependencies"` section.
5. Paste the `"@types/react"` entry under `"devDependencies"`.
6. Save the `package.json` file.

**Additional Steps:**

7. Run `npm install` to update the `package-lock.json` file with the changes made to `package.json`.

**Additional Note:**
- Moving `"@types/react"` to `devDependencies` ensures that it is only installed in the development environment, reducing the package size in production builds.
- This should be easy yea 😁 , feel free to ask question if you have any challenges following through the steps

### PR DESCRIPTION
**Pull request by DevYoma**
### Description
This PR removes the `@type/react` dependency from `dependencies` and adds it to the `devDependencies` in the projects `package.json` file

### Related Issue
Fixes #21 

### Screenshots/Screencasts
image

### Notes to Reviewer
I initially don't know what happened in the first commit. I ran `npm install` after moving `@types/react` dependency to the the devDependency. But I did it well in the second commit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +39,7 @@
     "@octokit/request": "^9.1.1",
     "@tailwindcss/container-queries": "^0.1.1",
     "@tailwindcss/typography": "^0.5.10",
-    "tailwindcss": "^3.4.1"
+    "tailwindcss": "^3.4.1",
+    "@types/react": "^18.2.69"
```
Comment (babblebey) [inline package.json:43]:
```suggestion
    "@types/react": "^18.2.69"
    "tailwindcss": "^3.4.1",
```

**Review by babblebey — APPROVED** [review]
This Looks Great @DevYoma 👍

File: package.json [hunk 2]
```diff
@@ -40,6 +39,7 @@
     "@octokit/request": "^9.1.1",
     "@tailwindcss/container-queries": "^0.1.1",
     "@tailwindcss/typography": "^0.5.10",
-    "tailwindcss": "^3.4.1"
+    "@types/react": "^18.2.69"
+    "tailwindcss": "^3.4.1",
```
Comment (babblebey) [inline package.json:43]:
```suggestion
    "@types/react": "^18.2.69",
    "tailwindcss": "^3.4.1"
```

Oops 🤭

**Review by babblebey — APPROVED** [review]
Great stuff, thank you @DevYoma 

### Quick note for next time 😉

- I renamed the PR title prefixing the word `fix(build):` to follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) standard. Look it up

**Comment by DevYoma** [comment]
I'm glad I could help :)
I'll take note of the corrections

**Pull request merged**